basics/github_desktop.md

```diff
@@ -0,0 +1,57 @@
+# GitHub Desktop
+
+A simple, yet powerful UI based tool for both git and GitHub that may be useful for all or part of your workflow.
+
+If you already have a git tool you like, or prefer command line tools, this may not be ideal for you. However,
+it is a great starting point if you are looking for a tool to allow you to easily get started with our GitHub Flow
+workflow.
+
+The [online documentation](https://docs.github.com/en/desktop) is more likely to remain up to date than anything we create here, so we'll just cover the
+basics of the sections likely to be most of use here.
+
+## Cloning
+
+The process of making a copy of a repository from GitHub (or any git source, but we'll focus on GitHub for the
+remainder of this document) to your local computer is called cloning.
+
+Cloning a repository is the first step in making local changes to files in a repository.
+
+[How to clone in GitHub Desktop](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-a-repository-from-github-to-github-desktop).
+
+## Syncing your local repo
+
+Before you make a branch, you'll always want to synchronize changes from GitHub as your local copies will not automatically stay synchronized.
+
+https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/keeping-your-local-repository-in-sync-with-github
```

Comment:
Could this link be titled with something human readable, rather than a bare URL?